### PR TITLE
The beginning

### PR DESCRIPTION
**Pull request by valevassallo**

### CODE REVIEW

**Review by stg101 — APPROVED** [review]

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+hi lmao
```
Comment (cristianbgp) [inline app/views/home/index.html.erb:1]: Why is this here? :thinking:

Comment (valevassallo) [reply app/views/home/index.html.erb:1]: It's temporary :joy:

**Review by cristianbgp — APPROVED** [review]
